device-registry/src/main/scala/org/genivi/sota/device_registry/DevicePackagesResource.scala

```diff
@@ -49,6 +57,9 @@ class DevicePackagesResource(namespaceExtractor: Directive1[AuthedNamespaceScope
           }
```

Comment:
The /devices/{:uuid}/packages route seems wrong to me, if we are using `deviceNamespaceAuthorizer` then it will check that the `uuid` is in the extracted namespace. But we are not sending a namespace since this request originate from the device, which doesn't know its namespace. I would prefer to have this in a /mydevice route which doesn't use the deviceNamespaceAuthorizer.